src/TimeSpans.jl

```diff
@@ -41,6 +41,11 @@ Return `TimeSpan(start(x), stop(x))`.
 """
 TimeSpan(x) = TimeSpan(start(x), stop(x))
 
+Base.in(x::TimePeriod, y::TimeSpan) = start(y) <= x <= stop(y)
+
+# work around <https://github.com/JuliaLang/julia/issues/40311>:
+Base.findall(pred::Base.Fix2{typeof(in), TimeSpan}, obj::Union{Tuple, AbstractArray}) = invoke(findall, Tuple{Function, typeof(obj)}, pred, obj)
```

Comment:
that's my first `invoke` (after seeing it in action in https://github.com/beacon-biosignals/Onda.jl/pull/68/commits/6b0afc0f2b5eb4ad2c20609961a23f63b8252e53), so let me know if I did it wrong. My other idea was to just do it manually,
```julia
Base.findall(pred::Base.Fix2{typeof(in), TimeSpan}, obj::Union{Tuple, AbstractArray}) = findall(x -> in(x, pred.x), obj)
```